assets/src/edit-story/components/canvas/pagepreview/index.js

```diff
@@ -114,6 +114,7 @@ function PagePreview({ index, gridRef, ...props }) {
 }
 
 PagePreview.propTypes = {
+  ariaLabel: PropTypes.string.isRequired,
```

Comment:
This doesn't seem to be required anymore since `aria-label` is used instead.

Reply:
ope. yup. good catch